subsys/socket_offload/providers/Kconfig.simplelink

```diff
@@ -0,0 +1,31 @@
+# Kconfig.simplelink - SimpleLink Socket Offload Provider configuration
```

Comment:
I'd say this should be treated as "socket offload driver", and thus live somewhere in drivers/